### PR TITLE
Add bootstrap accordion to FAQ page

### PR DESCRIPTION
**Pull request by superglu**
Update faq.html to incorporate css classes for a bootstrap accordion, and update the link from the sponsors page to link directly to a specific spot in it.

### CODE REVIEW

File: pybay/templates/frontend/faq_box.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
             </div>
             <div class="row">
                 <p class="more-link">
-                  <a href="{% url 'pybay_faq' %}">More FAQs</a>
+                  <a href="{% url 'pybay_faq' %}#sponsors">More FAQs</a>
```
Comment (epsy) [inline pybay/templates/frontend/faq_box.html:26]: This include is used on the frontpage and registration pages, so use a template variable for this. See https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#include

File: pybay/templates/frontend/faq.html [hunk 2]
```diff
@@ -7,24 +7,34 @@
...
-                                <div class="faq-item" id="faq-{{ faq.id }}">
-                                    <h3 class="lead blue-header"><strong>{{ faq.title }}</strong> <a class="glyphicon glyphicon-link" href="#faq-{{ faq.id }}"></a></h3>
-                                    <p>
-                                        {{ faq.body | safe }}
-                                    </p>
-                                </div>
-                            {% endfor %}
-                        </div>
+                    <div class="panel-group id=accordion">
+                                {% for slug, title, faqs in faq_categories %}
```
Comment (epsy) [inline pybay/templates/frontend/faq.html:11]: OPT: this is indented too far

**Review by epsy — CHANGES_REQUESTED** [review]
See in-line comments.
Has @grace-law signed off on the design?

For another review, we should auto-collapse on mobile.

**Comment by grace-law** [comment]
I can look at it after 7p today :)

Thanks everyone for the awesome contribution!

**Comment by PirosB3** [comment]
Can you make a PR towards master?

File: pybay/templates/frontend/faq.html [hunk 3]
```diff
@@ -7,24 +7,34 @@
...
+                                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse{{ slug }}">
+                                                    <h2 class="blue-header"><strong>{{ title }}</strong></h2>
+                                                </a>
+                                            </div>
+                                        {% endif %}
+                                            <div id="collapse{{ slug }}" class="panel-collapse collapse in">
+                                                <div class="panel-body">
+                                                    <div class="faq-items">
+                                                        {% for faq in faqs %}
+                                                            <a name="faq-{{ faq.id }}" style="position:relative;top:-60px">&nbsp;</a> {# The anchor with margin is just to get the page to scroll down a bit and not hide the title. #}
```
Comment (PirosB3) [inline pybay/templates/frontend/faq.html:24]: did you test this on mobile?

Comment (epsy) [reply pybay/templates/frontend/faq.html:24]: This is something Simeon added. If it needs to be changed, this must be done in a separate review.

**Pull request closed without merging**